Tasks/AzureFunction/task.json

```diff
@@ -0,0 +1,94 @@
+{
+	"id": "537FDB7A-A601-4537-AA70-92645A2B5CE4",
+	"name": "InvokeAzureFunction",
+	"friendlyName": "Invoke Azure Function",
+	"description": "Invoke Azure Function as a part of your process.",
+	"author": "Microsoft Corporation",
+	"helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=850348)",
+	"category": "Tool",
+	"preview": "true",
+	"visibility": [
+		"Release"
+	],
+	"runsOn": [
+		"Server"
+	],
+	"version": {
+		"Major": 0,
+		"Minor": 0,
+		"Patch": 1
+	},
+	"instanceNameFormat": "Invoke Azure Function: $(function)",
+	"groups": [],
+	"inputs": [{
+			"name": "function",
+			"type": "string",
+			"label": "Azure Function",
+			"defaultValue": "",
+			"required": true,
+			"helpMarkDown": "The Azure function Url for the function that needs to be invoked​."
+		}, {
+			"name": "key",
+			"type": "string",
+			"label": "Function Key",
+			"defaultValue": "",
+			"required": false,
+			"helpMarkDown": "The value for  an available function/host key for the function to be invoked. Advised to keep this secure using variables.​"
```

Comment:
Reword this to: - Function or Host key with which we can access this function. To keep the key secure, define a secret variable and use it here. Example: - $(myFunctionKey) where myFunctionKey is like this ZxPXnIEODXLRzYwCw1TgZ4osMfoKs9Zn6se6X/N0FnztfDvZbdOmYw==